atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -153,7 +152,9 @@ def _should_send(self, success, manual_canceled):
     def _get_image_name_and_repos(self):
 
         repos = []
-        dockerfile = df_parser(self.workflow.df_path, workflow=self.workflow)
+        dockerfile = self.workflow.build_dir.any_platform.dockerfile_with_parent_env(
```

Comment:
This will not work without initializing the build dirs

Reply:
Oh, good catch! That's why we need consistency /shakes_fist